policy/modules/system/systemd.te

```diff
@@ -379,6 +380,7 @@ files_list_usr(systemd_generator_t)
 fs_list_efivars(systemd_generator_t)
 fs_getattr_xattr_fs(systemd_generator_t)
 
+init_getattr_all_script_files(systemd_generator_t)
```

Comment:
`systemd-sysv-generator` will also _read_ all script files to determine the "init options" (https://github.com/SELinuxProject/refpolicy/commit/e63ad81ced3f8be92bb18b9d5f84388a04185819)